charts/argo-cd/templates/argocd-server/deployment.yaml

```diff
@@ -120,6 +120,9 @@ spec:
           failureThreshold: {{ .Values.server.readinessProbe.failureThreshold }}
         resources:
 {{- toYaml .Values.server.resources | nindent 10 }}
+      {{- if .Values.server.extraContainers }}
```

Comment:
Could we align the Helm template as per the indent above.

Reply:
It's done :-)